### PR TITLE
adding incident_key support for incidents command

### PR DESCRIPTION
**Pull request by roysagi**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/26048

## Description
Added a new argument to the incidents command to support getting an incident by de-dup key.
Also added the de-dup key to human-readable and context outputs for incident commands.
Also upgraded the README to fit our new standard.

### CODE REVIEW

File: Packs/PagerDuty/Integrations/PagerDuty/README.md [hunk 1]
```diff
@@ -1,1051 +1,784 @@
...
+
+    | **Parameter** | **Required** |
+    | --- | --- |
+    | API Key | True |
+    | Service Key (for triggering events only) | False |
+    | Trust any certificate (not secure) | False |
+    | Use system proxy settings | False |
+    | Fetch incidents | False |
+    | Incident type | False |
+    | Initial Fetch Interval(In minutes, used only for first fetch or after Reset last run) | False |
```
Comment (amshamah419) [inline Packs/PagerDuty/Integrations/PagerDuty/README.md:17]:
```suggestion
    | Initial Fetch Interval (In minutes, used only for the first fetch or after Reset last run) | False |
```

File: Packs/PagerDuty/Integrations/PagerDuty/PagerDuty.yml [hunk 2]
```diff
@@ -159,23 +168,32 @@ script:
...
     - default: false
-      description: Last date and time.  Using ISO 8601 Representation. E.g. PagerDutyIncidents until=2016-05-06T13:00Z
+      description: Last date and time.  Using ISO 8601 Representation. E.g. PagerDutyIncidents
+        until=2016-05-06T13:00Z
       isArray: false
       name: until
       required: false
       secret: false
+    - default: false
+      description: Incident de-duplication key,
```
Comment (Itay4) [inline Packs/PagerDuty/Integrations/PagerDuty/PagerDuty.yml:192]: can we add an example here?

File: Packs/PagerDuty/Integrations/PagerDuty/README.md [hunk 3]
```diff
@@ -1,1051 +1,784 @@
...
-      "last_status_change_by": "API Service",
-      "number_of_escalations": null,
-      "resolve_reason": null,
-      "resolved_by": "-",
-      "service_id": "P6UX4CI",
-      "service_name": "API Service",
-      "teams": [],
-      "urgency": "high"
+Use the PagerDuty integration to manage schedules and on-call users. 
+This integration was integrated and tested with PagerDuty API v2.
```
Comment (Itay4) [inline Packs/PagerDuty/Integrations/PagerDuty/README.md:2]: can we set here the product version and not the api version?

**Review by Itay4 — COMMENTED** [review]
please add the new argument to the test playbook

**Review by Itay4 — APPROVED** [review]

**Pull request merged**